docs/observability/aws/deploy-use-aws-observability/changelog.md

```diff
@@ -30,55 +48,45 @@ AWS Observability Solution (S3 Link for cloudformation template):`https://sumolo
 AWS Observability Solution (Terraform based) repo: [sumologic-solution-templates](https://github.com/SumoLogic/sumologic-solution-templates)
 
 New Features:
-
 * Support for Amazon EC2 CloudWatch Metric
 * Support for Amazon SNS service
 * New onboarding method via script for linux and windows
 
-Enhancements / Updates:
-
+Updates:
 * Updated “AWS Account Overview” and “AWS Region Overview” dashboards to monitor newly added Amazon SNS.
```

Comment:
```suggestion
* Updated “AWS Account Overview” and “AWS Region Overview” dashboards to monitor newly added Amazon SNS
```

Reply:
I'd suggest if you're writing complete clauses then put a period throughout the doc.